src/builder/possible_value.rs

```diff
@@ -133,7 +133,45 @@ impl PossibleValue {
     /// ```
     #[must_use]
     pub fn aliases(mut self, names: impl IntoIterator<Item = impl Into<Str>>) -> Self {
-        self.aliases.extend(names.into_iter().map(|a| a.into()));
+        self.aliases
+            .extend(names.into_iter().map(|a| (a.into(), false)));
+        self
+    }
+
+    /// Sets a *visible* alias for this argument value.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use clap::builder::PossibleValue;
+    /// PossibleValue::new("slow")
+    ///     .visible_alias("not-fast")
+    /// # ;
+    /// ```
+    #[must_use]
+    pub fn visible_alias(mut self, name: impl IntoResettable<Str>) -> Self {
```

Comment:
> Does not show aliases in error messages yet.

Whats holding that back?

Reply:
@epage Primarily, the amount of additional complexity required to pass groups of aliases through the error-context infrastructure. Secondarily, trying to figure out how to handle the "did you mean".

Reply:
So long as we hide visible aliases in the short help, we can forego them in the errors.

For the suggestion,
- If we consolidate the two possible value `ValueParser`s, we could probably switch the "did you mean" up a layer into those which could give greater control over it
- However, if we suggest a value that was not shown in the error, that might be confusing.

I can go either way

btw can you update the commit with some of these "for now" decisions so its easier for someone to discover them when making changes in the future?  People might discover the commit but its then a smaller set of people who might explore the PR comments.